backend/src/app.ts

```diff
@@ -7,6 +7,7 @@ import cookieParser from 'cookie-parser';
 import { checkToken } from './middleware/checkToken';
 import expressAsyncHandler from 'express-async-handler';
 import { tokenReader } from './middleware/tokenReader';
+import { presetsRouter } from './routers/presets';
```

Comment:
Not really a big problem, but you could consider importing this from /routers/index.ts like the other routers.